src/test/regress/input/multi_size_queries.source

```diff
@@ -0,0 +1,93 @@
+--
```

Comment:
We don't like .source files, they make harder to update regression files. Can we use one of the next two options?

i. Use existing tables
ii. Instead of copying from a file, use \copy. There are examples in other regression files.